src/Field.tsx

```diff
@@ -3,15 +3,33 @@ import * as React from 'react';
 
 import { FormikProps } from './formik';
 
+export interface FieldProps {
+  component: keyof React.ReactHTML | React.ComponentType<any>;
+  name: string;
+  type?: string;
+  value?: any;
+}
+
+export interface InputProps {
+  /** Name of the input */
+  name: string;
+  /** Classic React change handler, keyed by input name */
+  onChange: (event: React.ChangeEvent<any>) => void;
```

Comment:
ditto

Reply:
We only accept these 4 props for [field](https://github.com/jaredpalmer/formik/pull/186/files#diff-bc78961df770c0df68f6cd758a6f2afdR32)

Reply:
Confused as to why we need these InputProps at all? This is never exposed to the end user.

Reply:
Ahh I see this is used at the bottom of the file.